pkg/keeper/keeper.go

```diff
@@ -73,6 +74,7 @@ type scmProviderClient interface {
 	GetRepositoryByFullName(string) (*scm.Repository, error)
 	ListAllPullRequestsForFullNameRepo(string, scm.PullRequestListOptions) ([]*scm.PullRequest, error)
 	CreateComment(owner, repo string, number int, isPR bool, comment string) error
+	ToScmClient() *scm.Client
```

Comment:
This seems odd to have in the interface, or not?

Reply:
OK switched to the _scmProviderClient_ interface providing the API instead of go-scm's `Client` interface